home.html

```diff
@@ -121,157 +128,89 @@ <h5>Let's connect</h5>
             </div>
           </div>
         </section>
-        <section class="innerworkcontainer mt-4" id="about">
-          <div class="work project-1">
-            <div class="project">
-              <img src="assets/project1.png" alt="" />
-            </div>
-            <div class="description">
-              <h1>Tonic</h1>
-              <ul class="company">
-                <li class="item item1">
-                  <h3>Canopy</h3>
-                </li>
-                <li class="item item2">
-                  <h3>Back end dev</h3>
-                </li>
-                <li class="item item2">
-                  <h3>2015</h3>
-                </li>
-              </ul>
-
-              <ul class="tools">
-                <li class="item item1">HTML</li>
-                <li class="item item2">CSS</li>
-                <li class="item item2">Javascript</li>
-              </ul>
-
-              <p>
-                A daily selection of privately personalized reads, no accounts
-                or sign up required.
-              </p>
-
-              <div>
-                <a href="#" class="btn btn-resume mt-4">See the project</a>
-              </div>
-            </div>
-          </div>
-        </section>
-
-        <section class="innerworkcontainer mt-4">
-          <div class="work project-1">
-            <div class="description">
-              <h1>Multi-post stories</h1>
-
-              <ul class="company">
-                <li class="item item1">
-                  <h3>Facebook</h3>
-                </li>
-                <li class="item item2">
-                  <h3>Full stack dev</h3>
-                </li>
-                <li class="item item2">
-                  <h3>2015</h3>
-                </li>
-              </ul>
-
-              <ul class="tools">
-                <li class="item item1">HTML</li>
-
-                <li class="item item2">Ruby on rails</li>
-
-                <li class="item item2">CSS</li>
-                <li class="item item2">Javascript</li>
-              </ul>
 
-              <p>
-                Experimental content creation feature that allows users to add
-                to an existing story over the course of a day without spamming
-                their friends.
-              </p>
-              <div>
-                <a href="#" class="btn btn-resume mt-4">See the project</a>
-              </div>
-            </div>
-            <div class="project">
-              <img src="assets/project2.png" alt="" />
-            </div>
-          </div>
-        </section>
+        <div class="project-container" id="projects-container"></div>
```

Comment:
- [ ] [OPTIONAL] Fix this style issue when you are in screen size between desktop version and mobile version. (The text is out).

![image](https://user-images.githubusercontent.com/60164239/131576908-c87e6676-3948-4566-86b2-2514b110c1f3.png)